### PR TITLE
OutDegree and OutDegreeAv

### PR DESCRIPTION
**Pull request by c-yuwei**

### CODE REVIEW

File: braph2genesis/measures/_OutDegree.gen.m [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+%%%% ¡code!
+B = [
+    0   1   1
+    0   0   0
+    1   0   0
+    ];
+
+known_out_degree = {[2 0 1]'};
+
+g = GraphBD('B', B);
```
Comment (c-yuwei) [inline braph2genesis/measures/_OutDegree.gen.m:55]: i am not sure why here g = GraphBD('B', {B}) is not working....it is different from inDegree.

**Comment by egolol** [comment]
I cant compile this branch

**Comment by c-yuwei** [comment]
image
Hi @egolol In the attachment figure was the test results in my computer. Would you mind show me the error you got?

**Comment by c-yuwei** [comment]
The new form of the multiplex is implemented

**Review by annacanal — APPROVED** [review]

**Review by giovannivolpe — APPROVED** [review]

**Pull request merged**